uber/models.py

```diff
@@ -627,7 +627,7 @@ def match_to_group(self, attendee, group):
                 elif not matching:
                     return 'Badge #{} is a {} badge, but {} has no badges of that type'.format(attendee.badge_num, attendee.badge_type_label, group.name)
                 else:
-                    for attr in ['group', 'paid', 'amount_paid', 'ribbon']:
+                    for attr in ['group', 'group_id', 'paid', 'amount_paid', 'ribbon']:
```

Comment:
Not a bug per se, but one annoying thing that we occasionally run into with SQLAlchemy is the following

```python
>>> some_attendee.group = some_group
>>> some_attendee.group_id is None
True
```

In other words, setting the relationship doesn't immediately set the foreign key, which can put us in weird situations, so anytime we set one we should make a habit of setting the other.  (I personally consider this a design flaw in SQLAlchemy, though I understand why they don't automatically do this.)